pkg/controllers/success_policy.go

```diff
@@ -22,6 +22,107 @@ import (
 )
 
 // TODO: add unit tests for the functions in this file.
+// TestJobMatchesSuccessPolicy tests the jobMatchesSuccessPolicy function.
+func TestJobMatchesSuccessPolicy(t *testing.T) {
```

Comment:
In golang, unit tests are added with a corresponding *_test.go.

So I would move these tests to a pkg/controllers/success_policy_test.go file.